src/Android/Avalonia.Android/Avalonia.Android.csproj

```diff
@@ -6,6 +6,7 @@
     <MSBuildEnableWorkloadResolver>true</MSBuildEnableWorkloadResolver>
     <DebugType>portable</DebugType>
     <AndroidResgenNamespace>Avalonia.Android.Internal</AndroidResgenNamespace>
+    <IncludeAndroidSkia>true</IncludeAndroidSkia>
```

Comment:
I think these properties work only when build/HarfBuzzSharp.props is referenced directly in the project.